samba-integration-centos-ci-tests.sh

```diff
@@ -17,26 +17,31 @@ SCRIPT_GIT_BRANCH="centos-ci"
 SCRIPT_NAME="$(basename $0)"
 SCRIPT_PATH="$(realpath $0)"
 
-# enable additional sources for yum
-# (SCL repository for Vagrant, epel for ansible)
-yum -y install centos-release-scl epel-release
+# enable additional sources for yum:
+# - epel for ansible
+yum -y install epel-release
 
 # Install additional packages
 #
-# note: adding sclo-vagrant1-vagrant explicitly seems to fix
-#   issues where libvirt fails to bring up the vm with errors like this:
-#   "Call to virDomainCreateWithFlags failed: the CPU is incompatible with host
-#    CPU: Host CPU does not provide required features: svm" (or vmx)
-#
 yum -y install \
 	qemu-kvm \
 	qemu-kvm-tools \
 	qemu-img \
-	sclo-vagrant1-vagrant \
-	sclo-vagrant1-vagrant-libvirt \
 	git \
 	make \
-	ansible
+	ansible \
+	libvirt \
+	libvirt-devel
+
+yum -y group install "Development Tools"
```

Comment:
Why do we need to install "Development Tools" group?

Reply:
Needed in order to be able to do "vagrant plugin install"

Reply:
adding a comment